lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb

```diff
@@ -93,10 +93,10 @@ def mark_for_retry(task_id)
 
     def manageiq_env
       {
-        'api_token'      => Api::UserTokenService.new.generate_token(@workspace.ae_user.userid, 'api'),
-        'api_url'        => MiqRegion.my_region.remote_ws_url,
-        'workspace_guid' => @aw.guid,
-        'miq_group'      => @workspace.ae_user.current_group.description
+        'api_token'          => Api::UserTokenService.new.generate_token(@workspace.ae_user.userid, 'api'),
+        'api_url'            => MiqRegion.my_region.remote_ws_url,
+        'automate_workspace' => @aw.href_slug,
```

Comment:
@mkanoor - I'm behind on this change - but per other conversations I've picked up on: the `href_slug` method now includes the guid?

Reply:
Correct
```
irb(main):003:0> a.href_slug
=> "automate_workspaces/663bbf11-bffb-4b99-8874-380e60c8471e"
```